lib/LedgerSMB/Scripts/setup.pm

```diff
@@ -417,8 +417,8 @@ sub run_backup {
         die $request->{_locale}->text('Error creating backup file');
 
     if ($request->{backup_type} eq 'email') {
-        # suppress warning of single usage of $LedgerSMB::Sysconfig::...
-        no warnings 'once';
```

Comment:
Good call.
Seduced by a comment's promised ease, the primrose path I tread; @einhverfr , I'll follow your rede.
Barring side effects, I'll just delete the line.

Please explain how to find the commit to which you refer; my git skills are still weak.  The best I've got is to `git log -L:run_backup:setup.pm`  which stops when the file was presumably moved.